code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -393,6 +393,7 @@
 	for(var/mob/dead/observer/O in src.loc)
 		if(!O.client)	continue
 		if(O.mind && O.mind.current && O.mind.current.stat != DEAD)	continue
+		if(!(jobban_isbanned(O, "catban") || jobban_isbanned(O, "cluwneban")))	continue
```

Comment:
|| is ''or'', so you're basicly doing :
If person has been EITHER not catbanned or not cluwnebanned, continue.

It should be, if either is true, then stop the thing.

if(jobban_isbanned(O, "catban")) ||  jobban_isbanned(O, "cluwneban") return

If jobanned from either, return null.